packages/pwa/app/translations/README.md

```diff
@@ -89,6 +89,21 @@ For example, in your React component, you can add formatted messages like `intl.
 
 At the minimum, only defaultMessage is the required parameter. The message id is optional. If you don’t specify it, the id is auto-generated for you.
 
+### Creating message id
+
+Message ids is optional. If it is omitted, the ids will be generated automatically when you extract and compiled default messages.
+However, if you find that it is best to create message id manually, it is advised to include three elements as below order to provide the best context for translators 
```

Comment:
Btw, do we also want to update the file `CONTRIBUTING.md`? The ticket wanted us to update it.

Reply:
@vmarta and I had a chat about the id format. We will write down a document as a guideline for developers to construct the id and refer it from the contributing.md doc

Reply:
I'm wondering if our translation team has a more robust guideline that we could refer to?

Reply:
@kevinxh yes, they do, but we will also use it as a reference to write up our guideline because their doc looks like it is for the Business Manager system. Here is the [doc](https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=ENG&title=L10N%3A+Property+key+naming+rules).